frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeNameSection.tsx

```diff
@@ -13,6 +13,7 @@ const ServingRuntimeNameSection: React.FC<ServingRuntimeNameSectionProps> = ({ d
     <TextInput
       isRequired
       id="serving-runtime-name-input"
```

Comment:
Remove this id

Reply:
@christianvogt  is it advisory to replace id?

Reply:
Removing IDs is tricky. You must understand why the ID is present. IDs are often used for `aria` references. For example this ID is used by the `FormGroup` above as the `fieldId`:
> ID of an individual field or a group of multiple fields. Required when a role of "group" or "radiogroup" is passed in. If only one field is included, its ID attribute and this prop must be the same.

You should not remove this ID or others like it that are being referenced elsewhere.